enos/modules/test_cluster_health/main.tf

```diff
@@ -0,0 +1,35 @@
+# Copyright (c) HashiCorp, Inc.
+# SPDX-License-Identifier: BUSL-1.1
+
+terraform {
+  required_providers {
+    enos = {
+      source = "registry.terraform.io/hashicorp-forge/enos"
+    }
+  }
+}
+
+locals {
+  clean_token = trimspace(var.nomad_token) #Somewhere in the process, a newline is added to the token.
+}
+
+resource "enos_local_exec" "run_tests" {
```

Comment:
I think in order for this to have `verifies = quality.foo` fields, I think each of these scripts is going to need to be broken up into its own local exec: https://github.com/hashicorp/enos?tab=readme-ov-file#quality

Reply:
the qualities are defined in the enos scenario, this is just a terraform module that can be run on its own and only knows of qualities because I put them on the scripts, but wont do any extra verification other than what is in the code, if I understand correctly.

Reply:
Right, but unless we have a `verifies`, I don't think Enos has any way of knowing those qualities have been checked.

Reply:
We can follow-up on that in a subsequent PR though, I think.